lib/index.js

```diff
@@ -71,5 +71,23 @@ Sentry.prototype.loaded = function() {
  */
 
 Sentry.prototype.identify = function(identify) {
-  window.Raven.setUser(identify.traits());
+  window.Raven.setUserContext(identify.traits());
 };
+
+/**
+ * Clean out null values
+ */
+
+function reject(obj) {
```

Comment:
yeah it was on the server side. pretty simple function anyway and since already injected foldl might as well use it ;-p